xc7/yosys/synth.tcl

```diff
@@ -2,9 +2,7 @@ yosys -import
 
 # -flatten is used to ensure that the output eblif has only one module.
 # Some of symbiflow expects eblifs with only one module.
-synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp
-
-write_verilog $::env(OUT_SYNTH_V).premap.v
+synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp -iopad
```

Comment:
Why did you remove the premap output?

Reply:
I must have done that by accident. Fixed.